lib/routes/account.js

```diff
@@ -52,6 +52,7 @@ module.exports = function (
         var authSalt = crypto.randomBytes(32)
         var authPW = Buffer(form.authPW, 'hex')
         var locale = request.app.acceptLanguage
+        var userAgent = request.headers['user-agent']
```

Comment:
Should we sanitize this somehow? I can change my user-agent to whatever I want and inject silly strings into our token DB